contracts/libraries/TokenLib.sol

```diff
@@ -231,19 +230,19 @@ library TokenLib {
         address _module,
         uint256 _change,
         bool _increase,
-        address _polyToken,
+        IERC20 _polyToken,
```

Comment:
Note: could be `IPoly` to avoid casting on line 242,245 but I see that this is for consistency as polyToken is usually used as IERC20 without ever calling increase/decreaseApproval (which is only part of `IPoly`). Is probably cleaner the way it is right now keeping polyToken an IERC20 and only casting it to IPoly for increaseApproval :)